torch/csrc/jit/codegen/cuda/transform_replay.cpp

```diff
@@ -201,7 +201,7 @@ TensorDomain* TransformReplay::fullSelfReplay(
           new_self_root->getRootDomain(),
           new_rfactor_domain,
           new_domain,
-          new_self_root->contiguity());
+          self->contiguity());
```

Comment:
I wasn't sure if this is correct. Does `new_self_root` have an `rfactor` domain in this case?

Reply:
Yes, it can. It's just a self replay which I believe is used when replacing sizes with named sizes e.g. `T0->size(...)`